src/GSoft.AspNetCore.Authentication.ClientCredentialsGrant/ClientCredentialsDefaults.cs

```diff
@@ -0,0 +1,8 @@
+namespace GSoft.AspNetCore.Authentication.ClientCredentialsGrant;
+
+public static class ClientCredentialsDefaults
+{
+    public const string AuthenticationScheme = "ClientCredentials";
+    
+    public const string ClientCredentialsConfigSection = "Authentication:Schemes:ClientCredentials";
```

Comment:
I don't think this constant should be publicly exposed (or even exists);
* If devs use it, why not just create the section in appsettings.json + other config providers?
* If we only need it for tests, might not be worth it